cli-output/src/cli_output.rs

```diff
@@ -1684,6 +1684,8 @@ pub struct CliUpgradeableBuffer {
     pub address: String,
     pub authority: String,
     pub data_len: usize,
+    pub lamports: u64,
+    pub use_lamports_unit: bool,
```

Comment:
I think we should tag this field with `#[serde(skip_serializing)]` to avoid including it in the json obj